mantidproject/mantid
# Fixed various issues in the old Reflectometry GUI

As stated in the relevant issue, this PR fixes the following issues in the old Reflectometry GUI:
- Processing creates a _IvsQ workspace which is re-binned according to the setting, but we'd like to have the un-binned IvsQ workspace too and the re-binned one should be called _IvsQ_binned, like it used to.
- "Transfer" button doesn't work.
- "Save Workspaces" dialog doesn't populate the path correctly, it's missing a slash, e.g.: U:Skoda

**To test:**
- The GUI can be accessed from Interfaces->Reflectometry->ISIS Reflectometry
- Load runs using `INTER_NR_test2.tbl` from sample ISIS data.
- Ensure that processing runs creates a binned MomentumTransfer workspace "_IvsQ_binned" and an unbinned workspace "_IvsQ".
  - Ensure that it works for group workspaces (in which unbinned workspaces should also be grouped).
  - Go to Options->Refl Gui Options and uncheck "Use ReflectometryReductionOneAuto" Algorithm and check that processing runs produces the same result.
- Login to the catalog and enter `1520249` into 'RB Search' - the left column should be populated with runs. Select them all and click the transfer button - they should successfully transfer.
- Go to File->Save Workspaces - ensure that the save path is filled correctly (not missing /).



Fixes #18185.

*Release notes have been updated.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
